File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    var answer = '';
    
    arr = s.split(' ').map(Number)
    
    // console.log(arr)

    max = Math.max(...arr)
    min = Math.min(...arr)
    
    text = `${min} ${max}`
    
    return text
 
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String[] parts = s.split(" ");
        
        int[] arr = new int[parts.length];
        
        for(int i = 0; i < parts.length; i++) {
            arr[i] = Integer.parseInt(parts[i]);
        }
        
        Arrays.sort(arr);
            
        int min = arr[0];
        int max = arr[arr.length-1];
            
        String answer = min + " " + max;
        
        return answer;
    }
}